frontend/src/App.js

```diff
@@ -1,12 +1,32 @@
+import AppHeader from './components/AppHeader'
+import { Route, Switch } from 'react-router-dom'
+import Landing from './pages/Landing'
+import ArtistSearchResult from './pages/ArtistSearchResult'
+import styled from 'styled-components/macro'
 
-function App() {
+export default function App() {
   return (
-    <div>
-      <header>
-          Welcome to musiCompass!
-      </header>
-    </div>
-  );
+    <PageLayout>
+      <AppHeader/>
+      <Switch>
+        <Route exact path="/">
+          <Landing />
+        </Route>
+        <Route exact path="/artistsearch/:artistSearchTerm">
+          <ArtistSearchResult />
+        </Route>
+      </Switch>
+    </PageLayout>
+  )
 }
 
-export default App;
+const PageLayout = styled.div`
+  height: 100vh;
```

Comment:
you should keep in mind that when your content overflows the current view height, the styling will not be applied properly

Reply:
thx. really good point!